### PR TITLE
Adding slf4j-simple to resolve KPL output missing issue

### PR DESCRIPTION
**Pull request by Jackyjjc**
To resolve issue reported here: https://github.com/samcday/logstash-output-kinesis/issues/36 for logstash 2.x

Basically the same change as https://github.com/samcday/logstash-output-kinesis/pull/35/commits/1fd2cc60c4be89df48c733054560fa23f13991fa. However, logstash 2.x does not have log4j in its classpath so I included slf4j-simple instead. The downside is that there is no obvious way to change the log level for KPL so the output will be a bit noisy.

### CODE REVIEW

File: logstash-output-kinesis.gemspec [hunk 1]
```diff
@@ -32,6 +32,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-codec-json", "< 3.0.0"
   s.add_development_dependency "logstash-devutils"
   s.add_development_dependency "gem-release", "~>0.7.3"
+  s.add_development_dependency "rake", "< 10.2.0"
+  s.add_development_dependency "kramdown", "< 1.15.0"
```
Comment (Jackyjjc) [inline logstash-output-kinesis.gemspec]: https://github.com/samcday/logstash-output-kinesis/pull/35/commits/1fd2cc60c4be89df48c733054560fa23f13991fa

**Comment by Jackyjjc** [comment]
I've patched this to resolve the issue an internal team has. They are using the patch version already but I feel like I should submit the patch back to upstream in case anyone else is affected by this.

If you think 2.x is at the end of life and don't want to support it anymore then it is OK.

**Comment by asammut** [comment]
@samcday would it be possible to please get this PR merged :-)

**Comment by asammut** [comment]
Hi @samcday could we please get this PR merged ASAP. We are currently flying blind with our logstash implementation as many log lines are not making to the log files due to this.

**Comment by Jackyjjc** [comment]
@asammut I thought we are rolling out with the fork?

**Comment by asammut** [comment]
I was treating the fork more as a last resort. I don't quite understand why we wouldn't want to merge this and fix logging for everyone who uses this plugin

**Comment by samcday** [comment]
@asammut not sure if you're new to OSS, but here's a free tip - being demanding to open source maintainers will probably make them want to actively *not* help you :)

Being an ex-Atlassian I know there's still pockets of people/teams that have an allergic reaction to the idea of using / maintaining forks. Given the history I can totally understand that. But, treating a fork as a "last resort" arbitrarily is somewhat silly, for a couple of reasons:

 1) As I pointed out in my first comment, logstash 2.x series is very much EOL. If you were using a supported version of Logstash, you'd be using the latest version of this plugin which fixes the issue already.
 2) If you're so gung ho about this change, then it means you're confident in it, which should mean you'd have no qualms about running it as a fork for some period of time ;)

The reason I didn't rush out and merge this is I'm a little sketched out about adding slf4-simple to the classpath like this. Anyone else using Logstash 2.x and other plugins that also use slf4j may have already added slf4j-log4j or one of the other adapters to their classpath. Upgrading to this version might then cause their logging to go screwy.

Instead of lobbing slf4j-simple into this plugin, can't you simply add it to your own Logstash installation lib dir? AFAICT if you drop JARs into `$LSHOME/vendor/jruby/lib` they'll be added to the main Logstash classpath on boot-up.

**Comment by asammut** [comment]
@samcday sorry, we didn't want to add the overhead of maintaining a forked repo for this simple change.

I agree it's not ideal that we are not using the latest supported version of logstash but this is where my team is at. We aren't in a position to upgrade at the moment and that's not going to change any time soon.

In regards to the $LSHOME/vendor/jruby/lib suggestion, I have actually tried that solution and it didn't work for us.

**Comment by samcday** [comment]
Hi there, I'm archiving this repo (#51). I'm sorry for whatever protracted length of time I ignored your issue/PR for!

**Pull request closed without merging**